src/Draco.Compiler/Internal/Symbols/Metadata/MetadataMethodSymbol.cs

```diff
@@ -62,7 +62,14 @@ public override bool IsVirtual
                 || this.Override is not null;
         }
     }
+
     public override bool IsStatic => methodDefinition.Attributes.HasFlag(MethodAttributes.Static);
+
+    // TODO: Very hacky way of doing this
+    public override bool IsExplicitImplementation =>
+           this.Visibility == Api.Semantics.Visibility.Private
+        || this.Name.Contains('.');
```

Comment:
What is an explicit implementation

Reply:
Basically an interface member that gets hidden at the implementation site:
```cs
public class Foo : IList<int> {
    // other members...

    public IEnumerator<int> GetEnumerator() => /* ... */;

    // This is an explicit implementation
    IEnumerator IEnumerable.GetEnumerator() => /* ... */;
}
```

Reply:
Ah yes, I know what are thoses, I just didn't remembered about this thing when I've read the name.